dynamic-proxy/tests/graceful.rs

```diff
@@ -0,0 +1,62 @@
+use bytes::Bytes;
+use dynamic_proxy::body::to_simple_body;
+use dynamic_proxy::server::{HttpsConfig, SimpleHttpServer};
+use hyper::StatusCode;
+use std::convert::Infallible;
+use std::net::SocketAddr;
+use tokio::net::TcpListener;
+use tokio::time::Duration;
+
+// Ref: https://github.com/hyperium/hyper-util/blob/master/examples/server_graceful.rs
```

Comment:
Is this all copied in? Probably similar comment about MIT license, etc.

Reply:
This one isn't really copied in, it's just a useful example to refer to. Will add the other one to the license though.